docusaurus.config.js

```diff
@@ -77,6 +77,21 @@ module.exports = {
         {
           type: "docsVersionDropdown",
           position: "left",
+          dropdownItemsAfter: [
+            {
+              type: "html",
+              value: '<hr class="dropdown-separator">',
+            },
+            {
+              type: "html",
+              className: "dropdown-unsupported-versions",
+              value: "<b>Unsupported versions</b>",
+            },
+            ...["0.25"].map((version) => ({
+              label: version,
+              href: `https://unsupported.docs.camunda.io/${version}/`,
```

Comment:
I have the domain hardcoded here. I considered trying to make this dynamic based on which environment you're in, but because it's taking you off the current site I don't think it really matters.

Reply:
Hardcoded seems fine to me.